### ISSUE TITLE
Dúvida Geral

### ISSUE THREAD

**Issue by Militao36** [BoletoNet/boletonet#818]
Olá, sou novo aqui nesse projeto, estou tendo que implementar o Boleto na empresa onde trabalho, alguém que já implementou teria como me auxiliar? Preciso implementar de apenas 1 banco (Bradesco), porém os códigos que estão nos testes, quando vou usar no meu, alguns métodos não consta no mesmo.

**Comment by carloscds** [comment]
@Militao36 Você esta usando a ultima versao do componente?

**Comment by Militao36** [comment]
@carloscds  Então ainda estou na dúvida se uso a ultima versão ou a antiga, falaram que as duas estão funcionando, porém a versão antiga tem mais pessoas contribuindo.

**Comment by carloscds** [comment]
@Militao36 A versao atual esta no Nuget.

**Comment by Militao36** [comment]
@carloscds  Sim, porém estou tendo problemas para gerar, pois os códigos que estão nos testes, eu tento copiar alguns códigos e não rodam.

**Comment by carloscds** [comment]
@Militao36 Consegue enviar um exemplo de código ?

**Comment by Militao36** [comment]
@carloscds então eu consegui agora resolver esse erro, fiz de um modo diferente más aparentemente está funcionando normalmente agora, vou da uma olhada mais a fundo, qualquer dúvida volto perguntando aqui nesse issue, mas muito obrigado assim mesmo.

**Comment by Militao36** [comment]
@carloscds Opa a eu aqui novamente haha, más então consegui agora adicionar em meu projeto só que está dando um erro que não estou entendo 
`C#
                var str = new MemoryStream();
                var remessa = new ArquivoRemessa(objBoletos.Banco, TipoArquivo.CNAB240, 1);
                remessa.GerarArquivoRemessa(objBoletos, str);
                var arquivo = new FileStream(caminho + "/remessa.txt", FileMode.Open, FileAccess.ReadWrite);
                str.WriteTo(arquivo);
                arquivo.Close();
                str.Close();
`

Estou  usando esse código só que está lançado um erro que não e possível acessar um fluxo fechado, fiz algumas pesquisas tentei alterar um pouco o código porém sem sucesso.
